packaging/nnstreamer.spec

```diff
@@ -179,7 +179,9 @@ enable_tf=true
 enable_tf=false
 %endif
 
-meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir=%{_libdir} --bindir=%{nnstexampledir} --includedir=%{_includedir} -Dinstall-example=true -Denable-tensorflow=${enable_tf} %{api} -Denable-env-var=false -Denable-symbolic-link=false build
+%define enable_torch false
```

Comment:
[Question] enable_torch, is this always false in tizen? then add option -Denable-pytorch=false.

Reply:
it will be always false for now (till pytorch is supported in tizen)
Updated.